satpy/tests/reader_tests/test_ahi_hrit.py

```diff
@@ -15,8 +15,7 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
-"""The hrit msg reader tests package.
-"""
+"""The hrit msg reader tests package."""
```

Comment:
Typo: `jma` instead of `msg` (not your fault, it was there before)

Reply:
thanks, fixed (with `ahi` as the reader name)